storage/localstore/localstore.go

```diff
@@ -336,6 +342,50 @@ func New(path string, baseKey []byte, o *Options) (db *DB, err error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Create a index structure for storing pinned chunks and their pin counts
+	db.pinIndex, err = db.shed.NewIndex("Hash->pinCounter", shed.IndexFuncs{
```

Comment:
`Hash->PinCounter`

Reply:
You mean to change the 'p' to 'P'?

Reply:
Done

Reply:
I still see `Hash->pinCounter`. Please change to `Hash->PinCounter`